source/includes/admonition-wsl.rst

```diff
@@ -1,4 +1,4 @@
-.. warning:: Windows Subsystem for Linux (WSL) - Unsupported
+.. note:: Windows Subsystem for Linux (WSL) Support
 
 
-   MongoDB does not support the Windows Subsystem for Linux (WSL).
\ No newline at end of file
+   To run MongoDB in Windows Subsystem for Linux (WSL), refer to the `WSL documentation <https://docs.microsoft.com/windows/wsl/tutorials/wsl-database#install-mongodb>`.
```

Comment:
Just two things to change:

- Line break at 72 chars
- We need a double underscore after a link that goes outside of the docs site.

See the following suggestion:

```suggestion
   To run MongoDB in Windows Subsystem for Linux (WSL), refer to the
   `WSL documentation <https://docs.microsoft.com/windows/wsl/tutorials/wsl-database#install-mongodb>`__.
```